### PR TITLE
Add Ubuntu 20.04 support

### PR DESCRIPTION
**Pull request by davealden**
#### Pull Request (PR) description
This PR adds support for Ubuntu 20.04 LTS.

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
       "operatingsystemrelease": [
         "16.04",
         "18.04"
+        "20.04"
```
Comment (dhoppe) [inline metadata.json:34]: You also need to modify ```.sync.yml``` and ```.travis.yml```. Please keep in mind that Puppet Inc. does not provide any Puppet 5 packages for Ubuntu 20.04.

Comment (davealden) [reply metadata.json:34]: Ack - I had done a "grep -r 18.04" to look for what I needed to change (this is my first merge request in a long, long, long time :).  I didn't think to do a "grep -r 1804".  :).  I'll get it updated - thanks.

Comment (davealden) [reply metadata.json:34]: I've resubmitted with the updated .sync.yml and .travis.yml.  I see a "Resolve conversation" button - I'm going to guess that if it'll let me hit that button, I'm supposed to hit that button once I've made the change you requested?  :).

**Comment by jhawz** [comment]
Any idea when the next release will be? I need 20.04 support so just want to see if I should wait or fork. Thanks!

**Comment by bastelfreak** [comment]
Hi,
new release is coming in https://github.com/voxpupuli/puppet-fail2ban/pull/155

**Pull request merged**